### DISCUSSION TITLE
How to show dialog in dialog from bottom edge of the first dialog?

### DISCUSSION THREAD

**Discussion by rubymongoose** [quasarframework/quasar#17315] [General - Components / Directives / etc] [answered]
Hello, how can I show dialog in dialog with bottom position property, that will be shown from edge of the first dialog?

![Example](https://github.com/quasarframework/quasar/assets/52891428/9aa5b7b1-3f91-4106-8b21-0798b15cb863)

**Comment by ivanjaros** [comment, marked as answer]
dialog works like notification - with the window. this is not possible.
